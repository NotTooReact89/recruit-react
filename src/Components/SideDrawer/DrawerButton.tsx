import * as React from 'react';
import { IClickProps } from '..';
import './DrawerButton.css';

const DrawerButton = (props: IClickProps) => (
    <button className='toggle-button' onClick={props.click}>
        <div className='toggle-button__line' />
        <div className='toggle-button__line' />
        <div className='toggle-button__line' />
    </button>
);

export default DrawerButton;