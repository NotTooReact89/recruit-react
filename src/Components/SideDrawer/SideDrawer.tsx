import * as React from 'react';
import './SideDrawer.css';

const SideDrawer = (props: any) => (
    <div />
);

export default SideDrawer;