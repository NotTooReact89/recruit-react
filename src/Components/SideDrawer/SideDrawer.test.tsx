import * as React from 'react';
import SideDrawer from './SideDrawer';
const renderer = require('react-test-renderer');

it('should show side drawer correctly', () => {
  const tree = renderer.create((
    <SideDrawer show={true} />
  )).toJSON()
  expect(tree).toMatchSnapshot()
})

it('should hide side drawer correctly', () => {
    const tree = renderer.create((
      <SideDrawer show={false} />
    )).toJSON()
    expect(tree).toMatchSnapshot()
  })