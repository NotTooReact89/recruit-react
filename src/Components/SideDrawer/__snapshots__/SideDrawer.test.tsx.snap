// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should hide side drawer correctly 1`] = `
<nav
  className="side-drawer"
>
  <a
    className="back-button"
    href="/"
  >
    <img
      alt="back"
      height="50"
      src="back-icon.png"
      width="50"
    />
  </a>
  <ul>
    <li>
      <a
        href="/"
      >
        Menu Item 1
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Menu Item 2
      </a>
    </li>
  </ul>
</nav>
`;

exports[`should show side drawer correctly 1`] = `
<nav
  className="side-drawer open"
>
  <a
    className="back-button"
    href="/"
  >
    <img
      alt="back"
      height="50"
      src="back-icon.png"
      width="50"
    />
  </a>
  <ul>
    <li>
      <a
        href="/"
      >
        Menu Item 1
      </a>
    </li>
    <li>
      <a
        href="/"
      >
        Menu Item 2
      </a>
    </li>
  </ul>
</nav>
`;
